modules/node-tuning-operator.adoc

```diff
@@ -3,9 +3,19 @@
 // * scalability_and_performance/using-node-tuning-operator.adoc
 
 [id='about-node-tuning-operator-{context}']
-= Understanding the node tuning operator
+= About the node tuning Operator
 
-The node tuning operator helps you manage node-level tuning by orchestrating the
+The node tuning Operator helps you manage node-level tuning by orchestrating the
 tuned daemon.
 
-Here are more details about why it is used...
+The majority of high-performance applications require some level of kernel
+tuning. The node tuning Operator provides a unified management interface to
+users of node-level sysctls and more flexibility to add custom tuning specified
+by user needs. The Operator manages the containerized tuned daemon for
+{product-title} as a Kubernetes DaemonSet. It ensures custom tuning
+specification is passed to all containerized tuned daemons running in the
```

Comment:
Does the user know what a _containerized tuned daemon_ is?

Reply:
Does he need to?  If so, a reference to a github repo https://github.com/redhat-performance/tuned or a cross-reference to other RH docs?

Reply:
If you add a link, make sure it's to docs not a [github](https://github.com/openshift/openshift-docs/blob/enterprise-4.0/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites) repo.